### DISCUSSION TITLE
Can you make a unblocked vision?

### DISCUSSION THREAD

**Discussion by TerraNova-101** [SoftbearStudios/mk48#233] [Q&A] [answered]
My school block the original mk48, now I can only play mubai ver,can you make a unblocked version?

**Comment by finnbear** [comment, marked as answer]
Did you get your teacher's permission to play games? If so, you can try https://mk48.stratusflyer.com. If not, then realize they'll block that link too, and you should instead pay attention in class. Either way, follow all school and family policies, as required by our terms.

**Reply by TerraNova-101** [reply]
Thanks

**Reply by TerraNova-101** [reply]
ah i see history of myself
when i'm still an iPad kid
